elasticsearch/files/elasticsearch.yml

```diff
@@ -41,7 +41,9 @@ cluster.name: {{ server.cluster.name }}
 # from configuring them manually. You can tie this node to a specific name:
 #
 # node.name: "Franz Kafka"
-node.name: {{ server.get('name', '${HOSTNAME}') }}
+{%- if server.name is defined %}
+node.name: {{ server.name }}
```

Comment:
Do you really want to keep default as randomly generated names?
I would otherwise prefer to use ${HOSTNAME} as it was before (currently the change is not backward compatible as it will probably change current node names).